dashboard/templates/dashboard/vapid_validation.html

```diff
@@ -0,0 +1,19 @@
+{% extends "dashboard/base.html" %}
+
+{% block content %}
+    {% if not request.user.is_authenticated %}
+        {% include 'dashboard/includes/authenticate.html' %}
+    {% else %}
+        <p>Instructions for signing the token. Instructions for signing the token.  Instructions for signing the token. Instructions for signing the token.  Instructions for signing the token. Instructions for signing the token.  Instructions for signing the token. Instructions for signing the token.  Instructions for signing the token. Instructions for signing the token:</p>
+        <span id="jws-token">{{ jws_token }}</span>
+
+        <form id="vapid-validation" action="#TODO" method="POST">
+            {% csrf_token %}
+
+            <label for="{{ vapid_validation_form.signed_token.auto_id }}">Signed token:</label>
+            {{ vapid_validation_form.signed_token }}
+
+            <input type="submit" class="button" value="Validate">
+        </form>
+    {% endif %}
+{% endblock %}
```

Comment:
This UI looks really nice! We should definitely use it for the "Verify VAPID key" page, linked from the "Push Application Details" page ...